tools/ios/assemble_pod_package.py

```diff
@@ -81,16 +81,21 @@ def assemble_pod_package(
     pod_name = "onnxruntime-extensions-c"
     pod_summary = "ONNX Runtime Extensions C/C++ Pod"
     pod_description = "Pod containing pre and post processing custom ops for onnxruntime."
-    with open(framework_info_file, mode="r") as f:
-        framework_info = json.load(f)
-        ios_deployment_target = framework_info["apple_deployment_target"]
+    with open(xcframework_info_file, mode="r") as f:
+        xcframework_info = json.load(f)
+        ios_deployment_target = xcframework_info["iphonesimulator"]["apple_deployment_target"]
+        macos_deployment_target = ""
+        for key in xcframework_info.keys():
+            if "MacOSX" in key:
```

Comment:
hm, so why is it not "macosx"? seems odd that we can look up "iphonesimulator" fine.

is `CMAKE_OSX_SYSROOT` getting defined to "macosx"?

Reply:
from the building log, it looks like it's setting its value from apple.sysroot:
2023-12-16T01:17:53.6341750Z   /Users/runner/hostedtoolcache/Python/3.9.18/x64/bin/cmake /Users/runner/work/1/s -DPython_EXECUTABLE=/Users/runner/hostedtoolcache/Python/3.9.18/x64/bin/python -DOCOS_ENABLE_SELECTED_OPLIST=OFF -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO -DOCOS_BUILD_APPLE_FRAMEWORK=ON **-DCMAKE_OSX_SYSROOT=macosx** -DOCOS_ENABLE_SELECTED_OPLIST=ON -G 'Unix Makefiles' -DCMAKE_BUILD_TYPE=Release

not sure why in the resulting framework_info.json the cmake_OSx_ROOT is not equal to "macosx"..